WebRoot/h/changepass

```diff
@@ -63,6 +63,7 @@
     <c:set var="zimbraPasswordMinNumericChars" value="${mailbox.accountInfo.attrs.zimbraPasswordMinNumericChars[0]}"/>
     <c:set var="zimbraPasswordMinDigitsOrPuncs" value="${mailbox.accountInfo.attrs.zimbraPasswordMinDigitsOrPuncs[0]}"/>
     <c:set var="zimbraPasswordAllowedChars" value="${mailbox.accountInfo.attrs.zimbraPasswordAllowedChars[0]}"/>
+    <c:set var="zimbraPasswordAllowUsername" value="${not empty mailbox.accountInfo.attrs.zimbraPasswordAllowUsername[0] ? mailbox.accountInfo.attrs.zimbraPasswordAllowUsername[0] : FALSE}"/>
```

Comment:
will there be chances of zimbraPasswordAllowUsername being empty? isn't api going to provide default values always?

Reply:
this reminds me, classic ui code is same in cloud and zimbra 9
so are we going to implement this feature in both places together? if yes then we will need to backport backend changes for cloud